src/frontend/src/flows/recovery/confirmSeedPhrase.ts

```diff
@@ -0,0 +1,157 @@
+import { html, TemplateResult } from "lit-html";
+import { asyncReplace } from "lit-html/directives/async-replace.js";
+import { withRef, renderPage } from "../../utils/lit-html";
+import { Chan } from "../../utils/utils";
+import { ref, createRef, Ref } from "lit-html/directives/ref.js";
+import { mainWindow } from "../../components/mainWindow";
+import { I18n } from "../../i18n";
+
+import copyJson from "./confirmSeedPhrase.json";
+
+// A list of words, where "check" indicates if the user needs to double check (re-input) a word
+type Word = { word: string } & (
+  | { check: false }
+  | { check: true; elem: Ref<HTMLInputElement> }
+);
+
+// A list of indices nicely spread over the 25 words (anchor + 24 BIP39)
+export const checkIndices = [3, 7, 12, 13, 15, 17, 18, 20, 22, 24];
+
+const checkWord = (word: Word): boolean =>
```

Comment:
just fyi, I actually spent quite some time reviewing this particular function. I did not picked directly it was use to test if the input entry was valid. So if you want to add a small comment, review its name or make the return more obvious, not against.

Reply:
Ah my bad, that definitely deserves clarification